packages/edit-site/src/components/global-styles/screen-revisions/use-global-styles-revisions.js

```diff
@@ -24,49 +24,41 @@ const SITE_EDITOR_AUTHORS_QUERY = {
 const { GlobalStylesContext } = unlock( blockEditorPrivateApis );
 export default function useGlobalStylesRevisions() {
 	const { user: userConfig } = useContext( GlobalStylesContext );
-	const { authors, currentUser, isDirty, revisions, isLoading } = useSelect(
+	const { authors, currentUser, isDirty, revisions } = useSelect(
 		( select ) => {
 			const {
 				__experimentalGetDirtyEntityRecords,
 				getCurrentUser,
 				getUsers,
 				getCurrentThemeGlobalStylesRevisions,
-				isResolving,
 			} = select( coreStore );
 			const dirtyEntityRecords = __experimentalGetDirtyEntityRecords();
 			const _currentUser = getCurrentUser();
 			const _isDirty = dirtyEntityRecords.length > 0;
 			const globalStylesRevisions =
 				getCurrentThemeGlobalStylesRevisions() || [];
-			const _authors = getUsers( SITE_EDITOR_AUTHORS_QUERY );
+			const _authors = getUsers( SITE_EDITOR_AUTHORS_QUERY ) || [];
```

Comment:
Tiny nit (and it might not matter too much in this context): should we use a stable reference of an empty array, instead of creating a new empty array? I mostly noticed it because `authors` is used in the `useMemo` dependency array.

It might be worth using `EMPTY_ARRAY` two lines up, too?